src/routes/Dashboard/TaskEditForm.js

```diff
@@ -46,12 +56,12 @@ export default function TaskEditForm({
 
   React.useEffect(() => {
     if (id) {
-      getTodo(id);
+      getTask(id);
```

Comment:
One suggestion, can we please instead of using the name of service call `getTask()` we do `api.getTask()` ? In that way, we can easily distinguish a function that is created inside this component and a function that is imported from a module (in this case `api` module). 

Example:

``` javascript
import * as api from "../../api"
# some come here...
  React.useEffect(() => {
    if (id) {
      api.getTask(id);
# ... some code here
```

Reply:
Also, please do the error handling for 4xx and 5xx response. Again, we would ideally have this as a global error handling to keep it DRY.

Reply:
Please apply these to all api service callers on this component